src/counts.jl

```diff
@@ -231,22 +231,30 @@ end
 
 
 """
-    addcounts!(dict, x[, wv])
+    addcounts!(dict, x[, wv]; alg = :auto)
 
 Add counts based on `x` to a count map. New entries will be added if new values come up.
 If a weighting vector `wv` is specified, the sum of the weights is used rather than the
 raw counts.
+
+alg can be one of
```

Comment:
Use backquotes around object names and expressions. Use the Markdown bullet list syntax with `-` at the beginning of lines, and `:` instead to separate values from description. Use imperative rather than third person.